scripts/ipintutil

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+
+import netaddr
+import netifaces
+from natsort import natsorted
+from tabulate import tabulate
+
+from sonic_py_common import multi_asic
+from swsssdk import ConfigDBConnector, SonicDBConfig
+from utilities_common import constants
+from utilities_common import multi_asic as multi_asic_util
+
+
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
+
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        tests_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, tests_path)
+        import mock_tables.dbconnector
+        if os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] == "multi_asic":
+            import mock_tables.mock_multi_asic
+            mock_tables.dbconnector.load_namespace_config()
+except KeyError:
+    pass
+
+
+def get_bgp_peer():
+    """
+    collects local and bgp neighbor ip along with device name in below format
+    {
+     'local_addr1':['neighbor_device1_name', 'neighbor_device1_ip'],
+     'local_addr2':['neighbor_device2_name', 'neighbor_device2_ip']
```

Comment:
Suggest indenting one or three more spaces inside braces (2 or 4 total)

Reply:
Fixed in latest commit